packages/initializer/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "create-stryker",
```

Comment:
Since the package is called "create-stryker", lets call the directory name "create-stryker" as well.